docs/wrappers/peer_channel_postscript.md

```diff
@@ -137,6 +137,52 @@ Here's an example of the `peer channel join` command.
 
   You can see that the peer has successfully made a request to join the channel.
 
+### peer channel joinbysnapshot example
+
+Here's an example of the `peer channel joinbysnapshot` command.
+
+* Join a peer to the channel from a snapshot identified by the directory
+  `/snapshots/completed/testchannel/1000`. The snapshot was previously created on a different peer.
+
+  ```
+  peer channel joinbysnapshot --snapshotpath /snapshots/completed/testchannel/1000
+
+  2020-10-12 11:41:45.442 EDT [channelCmd] InitCmdFactory -> INFO 001 Endorser and orderer connections initialized
+  2020-10-12 11:41:45.444 EDT [channelCmd] executeJoin -> INFO 002 Successfully submitted proposal to join channel
+  2020-10-12 11:41:45.444 EDT [channelCmd] joinBySnapshot -> INFO 003 The joinbysnapshot operation is in progress. Check the status of joinbysnapshot to know if it is done.
```

Comment:
I suggest put the actual command to check the status in the response (of course, the command output needs to be updated, not just the generated doc).

```suggestion
  2020-10-12 11:41:45.444 EDT [channelCmd] joinBySnapshot -> INFO 003 The joinbysnapshot operation is in progress. Use "peer channel joinbysnapshotstatus" to check the status.
```